plugins/modules/aci_rest.py

```diff
@@ -537,5 +546,27 @@ def main():
     aci.exit_json(**aci.result)
 
 
+def process_rest_payload(payload, convert_values_to_str):
+    if convert_values_to_str:
+        if isinstance(payload, str):
+            payload = convert_payload_values_to_str(json.loads(payload))
+        else:
+            payload = convert_payload_values_to_str(payload)
+    return payload
+
+
+def convert_payload_values_to_str(data):
+    if data in [None, ""]:
```

Comment:
When a None value is provided should this be set to an empty string? In our collection for None we typically ignore the value being added to the payload, why diverge form this behaviour here?

This could unintentionally lead to values being overwritten on an existing MO to an empty string, which should be ignored. Like a description attribute for example.

Reply:
Also I do not understand why look for empty string, this is just a valid value that requires no translation

Reply:
I agree, I don't think we should be translating JSON Null to an empty string. Instead I think null values should be ignored in the request body. Although I am unsure if ACI actually accepts null json as a valid option anywhere.